src/app/home/events/events.component.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../../styles.colors';
+@import '../../../styles.variables';
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+  max-width: 1280px;
+  text-align: center;
+  display: grid;
+}
+
+.text {
+  color: $input-background;
+  padding-top: 40px;
+  font-size: 25px;
```

Comment:
Specific font sizes should be defined in styles.var or you should used the default ones provided by materialize